### PR TITLE
OTT-1168: Introduce SeatNonBid in hookoutcome (#pbs-3416)

### PR DESCRIPTION
**Pull request by ashishshinde-pubm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,8 +41,8 @@ inventory_url.yaml
 # generated log files during tests
 analytics/config/testFiles/
 analytics/config/xyz*
-analytics/build/xyz*
 analytics/filesystem/testFiles/
+analytics/build/xyz*
```
Comment (Pubmatic-Dhruv-Sonone) [inline .gitignore:45]: Remove this changes.

File: analytics/pubmatic/logger.go [hunk 2]
```diff
@@ -15,8 +15,10 @@ import (
 	"github.com/prebid/prebid-server/v2/hooks/hookexecution"
 	"github.com/prebid/prebid-server/v2/modules/pubmatic/openwrap/customdimensions"
 	"github.com/prebid/prebid-server/v2/modules/pubmatic/openwrap/models"
+	"github.com/prebid/prebid-server/v2/modules/pubmatic/openwrap/models/nbr"
 	"github.com/prebid/prebid-server/v2/modules/pubmatic/openwrap/utils"
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
+
```
Comment (Pubmatic-Dhruv-Sonone) [inline analytics/pubmatic/logger.go:21]: Nitpick: Remove blank line.

**Review by Pubmatic-Dhruv-Sonone — CHANGES_REQUESTED** [review]

File: endpoints/openrtb2/amp_auction.go [hunk 3]
```diff
@@ -279,8 +286,9 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 	var response *openrtb2.BidResponse
 	if auctionResponse != nil {
 		response = auctionResponse.BidResponse
+		seatNonBid.Append(auctionResponse.SeatNonBid)
+
```
Comment (Pubmatic-Dhruv-Sonone) [inline endpoints/openrtb2/amp_auction.go:290]: nitpick: Remove blank line

File: .gitignore (hunk 1 above)
Comment (ashishshinde-pubm) [reply .gitignore]: done

File: analytics/pubmatic/logger.go (hunk 2 above)
Comment (ashishshinde-pubm) [reply analytics/pubmatic/logger.go:21]: done

File: endpoints/openrtb2/amp_auction.go (hunk 3 above)
Comment (ashishshinde-pubm) [reply endpoints/openrtb2/amp_auction.go:290]: done

**Review by Pubmatic-Dhruv-Sonone — DISMISSED** [review]

**Review by Pubmatic-Dhruv-Sonone — APPROVED** [review]

**Review by pm-nilesh-chate — APPROVED** [review]

**Pull request merged**